### ISSUE TITLE
Browser Does Not Allow Access to All Menus/Menu items on Template/Menu Assignment Page

### ISSUE THREAD

**Issue by redders47** [joomla/joomla-cms#20446]
I am using Joomla 3.8.7. The php version is 7.0.28. I am using Windows 10 ![screen shot 2018-05-18 at 03 22 15](https://issues.joomla.org/uploads/1/9f994646f2055ce2bfb5a0491f423b99.jpg)![screen shot 2018-05-18 at 03 12 22](https://issues.joomla.org/uploads/1/3abec53414a5b26421caeb09e7502147.jpg)

My browser of choice is Chrome (the very latest version), but it now cannot properly access the page where Menus or menu items are assigned or unassigned to different templates using the Isis Back End Template.  I found after much trial and error that I could use the Hathor template successfully.

The attached photo shows my menus displayed across three columns.

Chrome recognises and allows me to toggle on and off in the first column only. I can toggle all menus/items on or off because the toggle is in this left hand column.  The ticks in middle and right hand columns were done before this issue arose.

I tried the Protostar Template in case my front end template was an issue – I get exactly the same result as with Chrome.

I tried Firefox - it allows me to toggle on and off in all three columns but I cannot access the account item at top right to log out etc.

I tried Edge - exactly the same result as Firefox.

I manage two different sites in two different accounts with Cloud Access - they cannot duplicate my issue at their end – but at my end both sites exhibit the same behaviour.

I have asked the Chrome Forum for help and followed all advice about removing extensions, checking for malware etc. No change to behaviour occurs. I have asked the Joomnla forum for advice without any result.

I wondered if I had a computer-specific issue. However, on my tablet, the behaviour is exactly the same with Chrome as on my PC.

I cannot put a time on when this went wrong for me but I assume now it was an upgrade to 3.8.5 or 3.8.7 that is causing the issue with the Isis Template. Note that the Hathor Template works fine for me - it lists my menus in only two columns and both columns are accessible with Chrome.

I have looked at all the options I can think of, and tried to eliminate various causes - I can only conclude on what I see, as a novice admittedly, that there is now some conflict between browsers and the Isis Template. Even though different browsers behave differently, not one of the three I tried allowed me to access the account icon at top right on the page concerned.

In Summary - I expect to be able to access and toggle on or off all menus and menu items. Using Chrome I cannot do that with the Isis Template. Other browsers provide more access than Chrome but none allow access to the top right hand account icon.

**Comment by ghost** [comment]
Thanks for Report. Similar but 4.0: #20377<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20446">issues.joomla.org/tracker/joomla-cms/20446</a>.</sub>

**Comment by Quy** [comment]
In `\administrator\templates\isis\css\template.css` remove lines 9091 & 9092 to confirm that this issue is fixed.

```
	-webkit-column-break-inside: avoid;
	-webkit-backface-visibility: hidden;
```

**Comment by redders47** [comment]
Thank you.

It fixes the menu assignment issue with Chrome.  However, the account item
top right remains inaccessible.  Obviously not a critical issue.



Richard

**Comment by Quy** [comment]
The account item should not be accessible because you are in edit mode. This is the same case when editing an article.

I will submit a PR by tomorrow for you to test. Thank you.

**Comment by redders47** [comment]
Hmmm.

What is a PR he says quietly.

I will be offline until about 9 June as well from tomorrow if that helps
with timing.

Richard

**Comment by Quy** [comment]
😄  PR stands for pull request that contains the code to fix the issue that needs to be tested by 2 people before it can be merged into the core package. You just need to test and mark your test result. I will provide instructions. Thanks again!

**Comment by redders47** [comment]
Okay.  Happy to be a part of this.  Learning all the time.

Richard

**Comment by Quy** [comment]
@redders47 Since you have already manually modified the changes, then please do the following to mark your test result:

* Open [Issue Tracker](https://issues.joomla.org/tracker/joomla-cms/20542)
* Log in with your Github account
* Click on blue `Test this` button above author's photo
* Select `Tested successfully`
* Click `Submit test result`<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20446">issues.joomla.org/tracker/joomla-cms/20446</a>.</sub>

**Comment by redders47** [comment]
Hi

Just done this.  I am really glad that, for a change, the problem wasn't ME.

Richard
